### PR TITLE
usb_device_check: added error checking in dmesg for linux guest

### PR DESCRIPTION
**Pull request by hereischen**
1. added error checking in dmesg for linux guest
2. reduced timeout for usb check tests

id: 1523014
Signed-off-by: Haotong Chen <hachen@redhat.com>

### CODE REVIEW

**Comment by hereischen** [comment]
@ngu-niny @maxujun Please help review, thanks.
PASS 1-smp_2.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.usb_device_check.max_devices_under_one_hub.nec-xhci

File: qemu/tests/usb_common.py [hunk 1]
```diff
@@ -129,6 +129,14 @@ def _verify_guest_usb():
                 return False
         return True
 
+    if params.get("os_type") == "linux":
+        logging.info("checking if there is I/O error in dmesg")
+        output = session.cmd_output("dmesg | grep usb",
```
Comment (ngu-niny) [inline qemu/tests/usb_common.py]: Talked with @maxujun, both uppercase and lowercase usb/USB should be considered.
And the same for the 'error' in line 137.

Comment (hereischen) [reply qemu/tests/usb_common.py]: @ngu-niny @maxujun  updated, thanks

**Comment by ngu-niny** [comment]
Test PASS:

'03-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.q35.qcow2.virtio_scsi.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.piix4-uhci' Finshed 2017-12-15 03:14:26 2017-12-15 03:16:05 PASS 99.61768
'04-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.q35.qcow2.virtio_scsi.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.ich9-uhci' Finshed 2017-12-15 03:16:06 2017-12-15 03:17:47 PASS 101.574765
'05-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.q35.qcow2.virtio_scsi.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.ich9-ehci' Finshed 2017-12-15 03:17:47 2017-12-15 03:19:29 PASS 101.661256
'06-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.q35.qcow2.virtio_scsi.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.nec-xhci' Finshed 2017-12-15 03:19:29 2017-12-15 03:21:09 PASS 100.160336
'07-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.q35.qcow2.virtio_scsi.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.qemu-xhci' Finshed 2017-12-15 03:21:09 2017-12-15 03:22:50 PASS 100.302894

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**